deploy/chart/templates/0000_50_olm_00-namespace.yaml

```diff
@@ -17,4 +19,6 @@ metadata:
     {{- if .Values.operator_namespace_psa }}
     pod-security.kubernetes.io/enforce: {{ .Values.operator_namespace_psa.enforceLevel }}
     pod-security.kubernetes.io/enforce-version: {{ .Values.operator_namespace_psa.enforceVersion }}
+    pod-security.kubernetes.io/audit: restricted
+    pod-security.kubernetes.io/warn: restricted
```

Comment:
@joelanford I _think_ we want these here as well, thoughts? CC @anik120

Reply:
+1

Reply:
Same comment as above.